### ISSUE TITLE
Rename `reserved_price` to `reservation_price` in Avellaneda MM

### ISSUE THREAD

**Issue by dennisocana** [hummingbot/hummingbot#4935]
Creating a task as mentioned in [this comment](https://github.com/CoinAlpha/hummingbot/pull/4873#issuecomment-993064873), to align to the paper more.

## Definition of Done

- [ ] Rename `reserved_price` to `reservation_price` in [avellaneda_market_making.pxd](https://github.com/CoinAlpha/hummingbot/blob/development/hummingbot/strategy/avellaneda_market_making/avellaneda_market_making.pxd) and [avellaneda_market_making.pyx](https://github.com/CoinAlpha/hummingbot/blob/development/hummingbot/strategy/avellaneda_market_making/avellaneda_market_making.pyx)

- [ ] Update the strategy documentation to reflect this change

**Comment by sisco0** [comment]
The _strategy documentation_ contained in already uses the _Reservation price_ term instead of _Reserved price_. This could be checked by using the next command:
```bash
grep -in "reserve" documentation/docs/strategies/avellaneda-market-making.md
```

**Compilation** using `./compile` and **testing** using `make test` operations have been carried on without inconvenience after changes.

**Comment by rapcmia** [comment]
Fixed on Hummingbot 1.1.0 release
